src/gpio.rs

```diff
@@ -279,7 +281,105 @@ where
     /// [`into_gpio_pin`]: #method.into_gpio_pin
     /// [`into_output`]: #method.into_output
     fn is_set_low(&self) -> Result<bool, Self::Error> {
-        Ok(!self.state.pin[T::PORT].read().port().bits() & T::MASK == T::MASK)
+        Ok(
+            !self.state.registers.pin[T::PORT].read().port().bits() & T::MASK
+                == T::MASK,
+        )
+    }
+}
+
+impl<'gpio, T, D> Pin<T, pin_state::Gpio<'gpio, D>>
+where
+    T: PinTrait,
+    D: direction::NotInput,
+{
+    /// Set pin direction to input
+    ///
+    /// This method is only available, if the pin is in the GPIO state and the
+    /// pin is not already in input mode, i.e. the pin direction is output or
+    /// unknown. You can enter the GPIO state using [`Pin::into_gpio_pin`].
+    ///
+    /// Consumes the pin instance and returns a new instance that is in output
+    /// mode, making the methods to set the output level available.
+    ///
+    /// # Example
+    ///
+    /// # TODO
```

Comment:
Errant `TODO` spotted. Maybe referring to the example, which is out of date?

Reply:
The example was for the gpio output, but it's out of date as well